### ISSUE TITLE
Training on multi classes

### ISSUE THREAD

**Issue by sainemustas321** [alexppppp/keypoint_rcnn_training_pytorch#16]
Hi, first of all. Thank you for sharing your excellent work. I tried to modify a code to train a multi classes detection.
In this line. Does it allow to change to my own multi classes??? thank you.
bboxes_labels_original = ['Glue tube' for _ in bboxes_original]

**Comment by WojtiF** [comment]
> Hi, first of all. Thank you for sharing your excellent work. I tried to modify a code to train a multi classes detection. In this line. Does it allow to change to my own multi classes??? thank you. bboxes_labels_original = ['Glue tube' for _ in bboxes_original]

Hi, did you find the way to solve this problem? I have an idea to solve two problems (object classification and keypoint detection) by two different algorithms,. But of course one would be better ;)

**Comment by MrZhang-YUBO** [comment]
> 

if you use two algorithms, the time would be influenced.

**Comment by WojtiF** [comment]
Of course, but it's my task to use neural network for classification and localization with orientation (I want to use keypoint detection for this). Even if it cause longer time.

The best option would be one algorithm but I can't find solution for this.
